### ISSUE TITLE
Trying to reproduce on IER-like dataset 

### ISSUE THREAD

**Issue by gautierevn** [tuyunbin/SCORER#5]
First thanks for this great work, it is very valuable to see your code and be able to run it,

I have reproduced your work on CLEVR-Change. Now I am trying to do it on a smaller dataset and as i see the results on Image Editing Request I think you could have some valuable insights if you are okay with sharing :)

1. Will you release the code for training on Image Editing Request ? Most of my questions could be answered with that I guess !
2. What was the training time with your GPU on Image Editing Request ?
3. What was the hyperparameters (learning rate, batch_size, other thing you can think of and you struggled with) ?

For now, i am training on a dataset of the size of IER. SCORER seems to struggle and I want to be sure I am not doing anything wrong to be able to compare my thing with it fairly.
 
Again, thanks for this work and for your time

**Comment by tuyunbin** [comment]
Hi, thank you for your high recognition to our work. I will organize and release the code for training on Image Editing Request. Besides, for the training time and other hyperparameters, I have already reported them in supp. https://openaccess.thecvf.com/content/ICCV2023/supplemental/Tu_Self-supervised_Cross-view_Representation_ICCV_2023_supplemental.pdf.

**Comment by gautierevn** [comment]
Thanks a lot for your answer, I'm looking forward to see your code and the supplementary material is great :)

**Comment by gautierevn** [comment]
Hi, I guess the most important files I need are just the train.py  and the dataset file suited for an IER training, if you have it somewhere I would be happy to take a look at it.

**Comment by gautierevn** [comment]
Right now the main issue is that it always output the same caption after few epochs.

**Comment by tuyunbin** [comment]
You can give me the email, and I will sent both files to you.&nbsp;


Yunbin&nbsp;Tu
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Gautier ***@***.***&gt;; 
发送时间: 2024年1月18日(星期四) 晚上11:31
收件人: ***@***.***&gt;; 
抄送: "Yunbin ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [tuyunbin/SCORER] Trying to reproduce on IER-like dataset  (Issue #5)





 
Hi, I guess the most important files I need are just the train.py  and the dataset file suited for an IER training, if you have it somewhere I would be happy to take a look at it.
